1.12/test/run

```diff
@@ -236,10 +236,18 @@ cid_file=$(mktemp -u --suffix=.cid)
 
 # Since we built the candidate image locally, we don't want S2I attempt to pull
 # it from Docker hub
-s2i_args="--pull-policy=never"
+s2i_build_args="--pull-policy=never --assemble-user=root"
```

Comment:
`root` isn't possible in OpenShift. So I'm not sure if we wants this...

Reply:
I kind of expected this. So can we put the file in app root and just symlink?

Reply:
> . So can we put the file in app root and just symlink?

That's one possible solution. Or we should be able to rewrite it, am I right?

Reply:
Yup, that sounds good as well: just let me know which solution I should do.

Reply:
It's up to you.

I prefer to rewrite file. But I'm not aware of any hard reason for this, so it's totally up to you.